operator/pkg/resources/cluster_role_binding.go

```diff
@@ -180,3 +198,32 @@ func (r *ClusterRoleBindingResource) RemoveSubject(
 
 	return nil
 }
+
+// RenderClusterRoleBinding renders a cluster role binding
+func RenderClusterRoleBinding(cluster *vectorizedv1alpha1.Cluster) *rbacv1.ClusterRoleBinding {
+	s := serviceAccountResourceKey(cluster)
+	return &rbacv1.ClusterRoleBinding{
+		// metav1.ObjectMeta can NOT have namespace set as
+		// ClusterRoleBinding is the cluster wide resource.
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      ClusterRoleBindingName,
+			Namespace: "",
+		},
+		TypeMeta: metav1.TypeMeta{
+			Kind:       ClusterRoleBindingKind,
+			APIVersion: RBACAPIVersion,
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      ServiceAccountKind,
+				Name:      s.Name,
+				Namespace: s.Namespace,
+			},
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: rbacv1.GroupName,
+			Kind:     ClusterRoleKind,
+			Name:     ClusterRoleBindingName,
```

Comment:
nit: This should refer to a variable named `ClusterRoleName`. As is it looks like it a mistake.